package.json

```diff
@@ -3,8 +3,9 @@
   "version": "1.0.0",
   "description": "Neomad styleguide, built with atomic design mindset",
   "scripts": {
-    "dev": "postcss ./content/styles/styleguide.css -o ./content/dist/styleguide.css --watch",
-    "prod": "postcss ./content/styles/main.css -o ./content/dist/main.css"
+    "lint": "stylelint content/styles/**/*.css --config postcss.linter.json",
+    "build-test": "postcss ./content/styles/main.css -o ./content/dist/neomad-test.css --env dev --watch --verbose",
+    "build-prod": "postcss ./content/styles/main.css -o ./content/dist/neomad-prod.css --env prod"
```

Comment:
I'd have called them `watch` and `build` by convention, but that's no big deal.

Reply:
let's go for this (also change in readme.md)